### ISSUE TITLE
sequence works well in editor but not work on mobile

### ISSUE THREAD

**Issue by tobytaro** [Demigiant/dotween#122]
I make a sequence, it works well in editor, but when I build my game to mobile (both ios and android), the sequence seems to execute all tweens inside at the same time. This is the code I wrote:
`public void CountDownAnimForAds() {

		Sequence mySequence = DOTween.Sequence();  

		mySequence.AppendInterval(0.5f);
		mySequence.Append(countDownImgList[0].DOFade(1f,0));
		mySequence.AppendCallback(()=> {
			AudioManager.instance.PlaySound("CountDown");
			Debug.Log("play sound 1");
		});

		mySequence.AppendInterval(0.5f);
	 	mySequence.Append(countDownImgList[0].DOFade(0,0.5f));

		mySequence.Append(countDownImgList[1].DOFade(1f,0));
		mySequence.AppendCallback(()=> {
			AudioManager.instance.PlaySound("CountDown");
			Debug.Log("play sound 2");

		});
		mySequence.AppendInterval(0.5f);

		mySequence.Append(countDownImgList[1].DOFade(0,0.5f));

		mySequence.Append(countDownImgList[2].DOFade(1f,0));
		mySequence.AppendCallback(()=> {
			AudioManager.instance.PlaySound("CountDown");
			Debug.Log("play sound 3");

		});
		mySequence.AppendInterval(0.5f);

		mySequence.Append(countDownImgList[2].DOFade(0,0.5f));
		mySequence.AppendCallback(()=> {
			GameManager.instance.Revive();
			SoundManager.ResumeAll();

		});

		mySequence.SetUpdate(true);
	}`

**Comment by tobytaro** [comment]
I found something very weird. I run CountDownAnimForAds() in the Unity Advertisement result callback, it has these problems, but if I run the same sequence in other place, it works well in both editor and mobile. 
I don't know why, pls help.

**Comment by Demigiant** [comment]
Hi,

Is it possible that you're creating the same Sequence twice for some reason? Also, when you say the Sequence all runs at the same time, you mean it complete immediately, or the tween play regularly but without any interval between them?

**Comment by tobytaro** [comment]
yes, i created almost the same sequence but in 2 different method, just the final AppendCallback() are different. I didn't use them at the same time.
the tween play regularly but without any interval between them
